app/form_models/jobseekers/account_feedback_form.rb

```diff
@@ -1,8 +1,12 @@
 class Jobseekers::AccountFeedbackForm
   include ActiveModel::Model
 
-  attr_accessor :origin, :comment, :rating
+  attr_accessor :comment, :email, :origin, :rating, :report_a_problem, :user_participation_response
```

Comment:
not particularly fond of the name `user_participation_response`.  These are all `responses`

Reply:
That was the name used on the general feedback form (which already had the question). I can change it to `research_participation` or `user_research_participation` perhaps?

Reply:
1 vote for `research_participation` if you change it

Reply:
Ok, I'll change the name of the field to `research_participation`, but in another PR!